src/components/overlayCookie/overlayCookie.scss

```diff
@@ -0,0 +1,28 @@
+@import '~theme/base/variables';
+@import '~theme/base/utils';
+
+#cookieWrapper {
+  width: 100%;
+  position: fixed;
+  bottom: 0;
+  background-color: $color-tertiary;
+  opacity: 0.9;
+  z-index: 99;
```

Comment:
This is the first time we are dealing with `z-index`, can you integrate something like this: https://jonsuh.com/blog/organizing-z-index-with-sass/ ?

Ideally we would want to have something like this:

```
// Put it in our theme folder

$z-layers: (
  "modal":   100,
  "tooltip": 200,
  "menu":    300,
  "alert":   400,
  "loading": 500,
);


// Returns the z-index value based on the layer's name
@function index($layer) {
  @return map-get($z-layers, $layer);
}
```

Then in this file:

```
@import '~theme/base/utils';

.cookieWrapper {
  z-index: index(alert);
}
```

It will make our lives easier later on!

Reply:
Good point!

Reply:
Done. FYI I also added a couple of negative styles, as they were used for backgrounds on the home page (I didn't investigate whether they were actually needed, just changed them).